sympy/rubi/tests/test_utility_function.py

```diff
@@ -270,3 +273,99 @@ def test_PositiveIntegerPowerQ():
 def test_FractionalPowerQ():
     assert FractionalPowerQ(a**2) == True
     assert FractionalPowerQ(a**sqrt(2)) == False
+
```

Comment:
okay, some functions like `FractionalPowerFreeQ` are not implemented yet. You can implement them in next PR.